### PR TITLE
fixes #11

### LINKED ISSUE #11: check of timeout value
In some cases, timeout could be set to 0 as well. It would be good to reconsider the check of initial timeout value.

### PR DESCRIPTION
**Pull request by KaterinaCZ**

### CODE REVIEW

File: test_timeout.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def test_elapsed(more_ms):
     assert elapsed >= more_ms and elapsed <= less_ms
 
 
-@pytest.mark.parametrize("wrong_timeout", (-3.1, -1, 0.0, 0, 1.0, 1e10))
+@pytest.mark.parametrize("wrong_timeout", (-3.1, 0.0, 1.0, 1e10))
```
Comment (Slavaqq) [inline test_timeout.py]: Why don't we have to test negative int anymore?

Comment (Slavaqq) [reply test_timeout.py]: Yes, I understand this but we should still test the situation that someone add this option for an example `-1`. It won't be part of this test function.

Comment (KaterinaCZ) [reply test_timeout.py]: Please, check the matter in the new commit.

File: test_timeout.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 
 TIMEOUTS_MS = (2, 10, 50, 100, 200, 1000, 1500, 5000, 20000)
+NEGATIVE_TIMEOUTS_MS = (-100, -10, -1, 0)
```
Comment (Slavaqq) [inline test_timeout.py]: Zero is not a negative number.

Comment (KaterinaCZ) [reply test_timeout.py]: I know, but `JUST_SOME_OTHER_TIMEOUTS_WE_WANT_TO_TEST_MS` didn't seem quite right either and no better alternative crossed my mind yesterday. What about `POSITIVE_TIMEOUTS_MS` and `TIMEOUTS_MS` instead? That would be more accurate.

File: test_timeout.py [hunk 3]
```diff
@@ -60,3 +61,15 @@ def test_remaining_type(timeout_ms):
 def test_elapsed_type(timeout_ms):
     t = Timeout(timeout_ms)
     assert type(t.elapsed()) is int
+
+
+@pytest.mark.parametrize("timeout_ms", NONPOSITIVE_TIMEOUTS_MS)
+def test_remaining_negative_value(timeout_ms):
```
Comment (Slavaqq) [inline test_timeout.py]: Change the names of the functions

**Pull request merged**